src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,91 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int RESIZE_COEFFICIENT = 2;
+    private int threshold;
+    private int size;
+    private Node<K, V>[] bucketsArray;
```

Comment:
```suggestion
    private Node<K, V>[] buckets/table/data; 
```
давай не будемо використовувати слово Array, і так зрозуміло що це масив